### ISSUE TITLE
Name, role and state properties are not defined for "Advanced" expand/collapse control present under 'New collection' blade.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1015]
Note: User credentials should NOT be included in the bug. 
Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Add Collection" Menu item/Button and select it.
Navigate through all the controls under add Collection/New collection blade.
Navigate to "Advanced" expand/collapse control present under 'New collection' blade.
Press F12 key to open the Developers Tool and inspect the code.
Verify the issue.

Actual Result:
Name, role and state properties are not defined for "Advanced" expand/collapse control present under 'New collection' blade.

Expected Result:
Name(Advanced), role(button) and state(aria-expanded) properties should be defined for "Advanced" expand/collapse control present under 'New collection' blade so that the screen reader passes proper information about the controls to the end user on navigation.

Note:
This issue also exists for similar controls present under 'New Graph' blade of Data explorer screen for Created cosmos DB  after selecting "Gremlin(graph)" option.

![image](https://user-images.githubusercontent.com/8519407/132721602-3e2eb468-6c1d-45ab-bc86-49f5e2029589.png)

**Comment by ato9000** [comment]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1216117
